libs/pbd/fpu.cc

```diff
@@ -316,9 +304,21 @@ FPU::FPU ()
 #endif
 		}
 
+		/* Get CPU extended features: Like AVX512F */
+		if (num_ids > 0) {
```

Comment:
This check is redundant. This can only be reached if `num_ids > 0`

Reply:
Fixed!